Rakefile

```diff
@@ -55,3 +61,9 @@ task :task_6648_generate_exit_code, [:pathToVBOXConfigFile, :pathToMDBCIConfigFi
 task :task_6755_show_platform_versions do |t| RakeTaskManager.new(t).run end
 task :task_6844_ssh_pty_bug do |t| RakeTaskManager.new(t).run end
 task :task_6754_bug do |t| RakeTaskManager.new(t).run end
+task :task_6783_show_boxes do |t| RakeTaskManager.new(t).run end
+task :task_6813_divide_show_boxes do |t| RakeTaskManager.new(t).run end
+task :task_node_product do |t| RakeTaskManager.new(t).run end
+task :task_boxes_manager do |t| RakeTaskManager.new(t).run end
+task :task_repos_manager do |t| RakeTaskManager.new(t).run end
+task :task_session do |t| RakeTaskManager.new(t).run end
```

Comment:
Please divide parametrized and nonparametrized tests in the bottom of this file by comments/